### PR TITLE
Immutable support for payload reducer

### PR DESCRIPTION
**Pull request by belkosyak**

### CODE REVIEW

File: src/withPayloadReducer.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
+import { identity } from 'lodash';
```
Comment (xamfoo) [inline src/withPayloadReducer.js]: To keep the library lightweight, we import individual functions from lodash as a convention. What do you think?

``` js
import identity from 'lodash/identity';
```

**Comment by belkosyak** [comment]
agree, fixed

File: src/withPayloadReducer.js (hunk 1 above)
Comment (belkosyak) [inline src/withPayloadReducer.js]: agree, fixed

File: README.md [hunk 2]
```diff
@@ -135,11 +135,15 @@ const objectReducer = withMapStateReducer(
...
+withPayloadReducer(
+  actionType: String,
+  mapResult: ?(resultBefore: any) => resultAfter
+): (reducer) => reducer
 ```
 
-Creates a second-order reducer which returns the payload of the action
-for the given action type.
+Creates a higher-order reducer which returns the payload of the action
+for the given action type. Before returning it maps result `mapResult` function, which
```
Comment (xamfoo) [inline README.md]: Minor edit.

... it maps the payload with `mapResult` function, ...

Comment (belkosyak) [inline README.md]: fixed

**Pull request merged**